control_panel_api/tests/test_aws.py

```diff
@@ -0,0 +1,65 @@
+from unittest.case import TestCase
+from unittest.mock import patch
+
+from control_panel_api import aws
+
+
+@patch('boto3.client')
+class AwsTestCase(TestCase):
+    def test_create_bucket(self, mock_client):
+        aws.create_bucket('bucketname', 'eu-west-1')
+        mock_client.return_value.create_bucket.assert_called()
+
+    def test_delete_bucket(self, mock_client):
+        aws.delete_bucket('bucketname')
+        mock_client.return_value.delete_bucket.assert_called()
+
+    def test_put_bucket_logging(self, mock_client):
+        aws.put_bucket_logging('bucketname', 'target_bucket', 'target_prefix')
+        mock_client.return_value.put_bucket_logging.assert_called()
+
+    def test_create_policy_json_encoded(self, mock_client):
+        aws.create_policy('bucketname', {'foo': 'bar'})
+        mock_client.return_value.create_policy.assert_called_with(
+            PolicyName='bucketname',
+            PolicyDocument='{"foo": "bar"}'
+        )
+
+    def test_delete_policy(self, mock_client):
```

Comment:
One thing I try to do when doesn't make things incredibly annoying to test is to only tests the public interface of a module/class (which is your "contract" with the client code).

In this specific module maybe should we test only "public" functions not internal ones? This is more a generic thing so I'm not saying to change it.

The reason why usually makes sense to test only the public functions is that then you don't "break" the tests every time you refactor/rename internal/utility functions. (don't know if this make sense). E.g. you may decide to ditch the `delete_policy()` function or rename it to `_delete_policy()` or change some params and the tests would only cover the public functions and the expected side effects.

**BUT** as I said, I'm not saying to change this right now, it's just a general comment.